src/common/film.c

```diff
@@ -395,6 +395,10 @@ void dt_film_remove(const int id)
   sqlite3_stmt *stmt;
 
   gboolean remove_ok = TRUE;
+   
+  // explicitly enable foreign keys
```

Comment:
This is already done globally when initiating the db connection in dt_database_init() so probably not needed here.